drakrun/drakrun/injector.py

```diff
@@ -52,22 +62,40 @@ def _get_cmdline_createproc(self, exec_cmd: str, wait: bool = False) -> List[str
     def write_file(
         self, local_path: str, remote_path: str, timeout: int = 60
     ) -> subprocess.CompletedProcess:
-        """Copy local file to the VM"""
-        injector_cmd = self._get_cmdline_writefile(local_path, remote_path)
+        """
+        Copy local file to the VM
+        we pass (timeout-5) to drakvuf to give it 5 seconds to finish it's loop
+        """
+        drakvuf_timeout = timeout - 5 if timeout != 0 else 0
```

Comment:
```suggestion
        drakvuf_timeout = timeout - 5 if timeout > 5 else 0
```
shouldn't it look like this?